jumpscale/packages/stellar_stats/bottle/stellar_stats.py

```diff
@@ -49,4 +49,49 @@ def get_stats():
     return results
 
 
+@app.route("/api/total_tft")
+def total_tft():
+    query_params = request.query.decode()
+    network = query_params.get("network", "public")
+    tokencode = query_params.get("tokencode", "TFT")
+
+    # cache the request in local redis
+    redis = j.clients.redis.get("redis_instance")
+    cached_data = redis.get(f"{network}-{tokencode}-total_tft")
+    if cached_data:
+        return cached_data
+
+    collector = StatisticsCollector(network)
+    stats = collector.getstatistics(tokencode, False)
+
+    total = stats["total"]
+    redis.set(f"{network}-{tokencode}-total_tft", total, ex=600)
+    return f"{total}"
```

Comment:
we want to return a float value to the user as described in the issue

```suggestion
    return total
```

Reply:
This cause 500 error